pkg/diagnostics/cluster/aggregated_logging/serviceaccounts.go

```diff
@@ -0,0 +1,43 @@
+package aggregated_logging
+
+import (
+	"fmt"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/util/sets"
+)
+
+var serviceAccountNames = sets.NewString("logging-deployer", "aggregated-logging-kibana", "aggregated-logging-curator", "aggregated-logging-elasticsearch", fluentdServiceAccountName)
+
+const serviceAccountsMissing = `
+Did not find any logging ServiceAccounts.  The logging infrastructure may not be able to
+properly function without this account which receives cluster permissions.  You may need to 
+re-run the installer.
+`
+const serviceAccountMissing = `
+Did not find ServiceAccount '%s'.  The logging infrastructure may not be able to
```

Comment:
The logging infrastructure may not function properly without a service account with cluster permissions.